data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1680 @@
+			`	It thinks for a moment, then answers, "No. We never intended to fight. It was the Saryds, Kimek and Arachi of old who fought us. We are not a warring people."`
+			`	Seemingly satisfied with the answer, Oobat moves on to the next. She questions the Quarg for about ten minutes on their activities outside the Coalition, their relation with the humans and other species, even having you move the camera around for a bit to show the local human-Quarg interactions, as the Quarg compares dealing with humanity to dealing with the Coalition species. Oobat moves from question from question as if in a checklist, until she arrives at the last one. "About finished, we are. Thankful for your answers I am, just one last thing, to tranquilize the people back home. Interested in returning to take back the rings, are you?"`
+			`	"Of course," it says without hesitation. "The rings are our homes. My home." Clearly not expecting that answer, Oobat looks at you in confusion.`
+			choice
+				`	"It's been thousands of years. You're still lingering on those rings?"`
+				`	"If you want them back, why haven't you returned by now?"`
+					goto why
+			`	"Lingering on... 'those' rings? Our rings. Our homes," it replies. "Do you so easily forget yours?" The Quarg crosses one arm, grabbing its shoulder, lowering its head as its back curves. Patches of skin start turning pitch black, and it brings its other arm to its head as if in pain.`
+				goto accept
+			label why
+			`	"Because that would have forced a fight. The three... these Heliarchs, would not listen. They never listened," it answers, crossing one arm and grabbing its shoulder, lowering its head as its back curves. Patches of skin start turning pitch black, and it brings its other arm to its head as if in pain.`
+			label accept
+			`	By now the atmosphere's changed, with no humans in sight around you, and dozens more Quarg forming circles around your ship, eyes fixated on Oobat. The ones closest to you watch the one who was being interviewed carefully, like waiting for some sign, as its whole body shifts color. Oobat is still looking at you, confused and, now, worried.`
+			choice
+				`	"I think that does it for the interview."`
+				`	"We should leave. Now."`
+			`	She nods, and gives a quick thank to the Quarg again, turning to go back inside your ship. The Quarg stops her, though, reaching out one arm quickly to grab one of her legs. Slowly, its skin turns back to the grayish blue from before, as it calms down. "Should you succeed, please, permit me return home." The Quarg around you disperse, the one who was interviewed standing up again and moving back to other sections of the station.`
```

Comment:
is that Quarg 6000 years old? 
also usually when you come to a Quarg world anyone of them is capable of answering any questions you may have? I'm not sure about this but it's the impression I had